pkg/cmd/create/cluster/scenario/create/hub/common/machinepool_cr.yaml

```diff
@@ -10,8 +10,14 @@ spec:
     name: "{{ .managedCluster.name }}"
   name: worker
   platform: 
+{{ if (eq .managedCluster.cloud "openstack") }}
+    openstack:
+      flavor: {{ .managedCluster.openstack.workerFlavor }}
```

Comment:
Can you reference the value set in the install-config for the flavor as it is done for the replica below?

Reply:
No, I can't because the key is different.  type vs flavor

Reply:
then sorry but you will need to set the same default as in the install_config.yaml